services/gateway/apis/http-version3/swagger/definitions/events.json

```diff
@@ -1,92 +1,92 @@
-                "type": "string",
-                "example": "token",
-                "description": "module name of the transaction.\n"
-            },
-            "data": {
-                "type": "object",
-                "properties": {
-                    "amount": {
-                        "type": "string",
-                        "example": "15000000"
-                    },
-                    "recipientAddress": {
-                        "type": "string",
-                        "format": "address",
-                        "example": "lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y99",
-                        "description": "The Lisk Address is the human-readable representation of a blockchain account.\nIt is 41 character long identifier that begins with `lsk`."
-                    },
-                    "data": {
-                        "type": "string",
-                        "example": "test"
-                    }
-                }
-            },
-            "block": {
-                "type": "object",
-                "properties": {
-                    "id": {
-                        "type": "string",
-                        "format": "id",
-                        "minLength": 1,
-                        "maxLength": 64,
-                        "example": "01967dba384998026fe028119bd099ecf073c05c045381500a93d1a7c7307e5b",
-                        "description": "Unique identifier of the block.\nDerived from the block signature.\n"
-                    },
-                    "height": {
-                        "type": "integer",
-                        "minimum": 1,
-                        "example": 8350681,
-                        "description": "The height of the block in which the transaction was included.\n"
-                    },
-                    "timestamp": {
-                        "type": "integer",
-                        "example": 1613323667,
-                        "description": "UNIX Timestamp"
-                    }
-                }
-            },
-            "topics": {
-                "type": "array",
-                "items": {
-                    "type": "string",
-                    "example": "478842a844914f18a1c620a6494bf9931d0a862e96212bf5fc6f3ca18658febe"
-                }
-            }
-        }
-    },
-    "eventsWithEnvelope": {
-        "type": "object",
-        "required": [
-            "data",
-            "meta"
-        ],
-        "properties": {
-            "data": {
-                "description": "List of events",
-                "type": "array",
-                "items": {
-                    "$ref": "#/definitions/events"
-                }
-            },
-            "meta": {
-                "$ref": "#/definitions/Pagination"
-            }
-        }
-    }
+	"events": {
+		"type": "object",
+		"required": [
+			"moduleID",
+			"moduleName",
+			"topics",
+			"data"
+		],
+		"properties": {
+			"moduleID": {
+				"type": "string",
```

Comment:
May be in separate PR but response for events also need to be updated.

Reply:
Updated!